drivers/adxl345/adxl345.c

```diff
@@ -34,7 +34,8 @@
 #define BUS                 (dev->params.i2c)
 #define ADDR                (dev->params.addr)
 
-int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
+int adxl345_init(adxl345_t *dev, const adxl345_params_t* params, uint8_t range,
+                 uint8_t rate, uint8_t full_res, float scale_factor)
```

Comment:
Sorry I didn't remember that part ... I'll change that but I saw others drivers with more than `dev `and `params` as arguments like `drivers/isl29020`